src/ConfigWidgets/SymbolsDialogTest.cpp

```diff
@@ -39,12 +39,19 @@ class MockPersistentStorageManager : public orbit_symbol_paths::PersistentStorag
               LoadModuleSymbolFileMappings, (), (override));
 };
 
+void SetUpExpectMockCalls(MockPersistentStorageManager* mock_storage_manager,
+                          std::vector<std::filesystem::path> load_paths = {},
+                          absl::Span<const std::filesystem::path> save_paths = {}) {
+  EXPECT_CALL(*mock_storage_manager, LoadPaths).WillOnce(testing::Return(std::move(load_paths)));
+  EXPECT_CALL(*mock_storage_manager, SavePaths)
+      .WillOnce([save_paths](absl::Span<const std::filesystem::path> paths) {
+        EXPECT_EQ(paths, save_paths);
+      });
+}
+
 TEST(SymbolsDialog, ConstructEmpty) {
   MockPersistentStorageManager mock_storage_manager;
-  EXPECT_CALL(mock_storage_manager, LoadPaths())
-      .WillOnce(testing::Return(std::vector<std::filesystem::path>{}));
-  EXPECT_CALL(mock_storage_manager, SavePaths)
-      .WillOnce([](absl::Span<const std::filesystem::path> paths) { EXPECT_TRUE(paths.empty()); });
+  SetUpExpectMockCalls(&mock_storage_manager);
```

Comment:
This makes the test less understandable because you don't see anymore what it is actually testing. I think a test fixture would be more appropriate because it could have member functions that have names and make it easier to understand. For example `void SetLoadPaths(...)` and `void SetExpectedSavePaths(...)`

Reply:
used a fixture